Update guide / service pages (#17)
mesg-foundation/docs: guide/service/deploy-a-service.md

mesg-core service deploy PATH_OF_THE_SERVICE
```

> This will generate a Service ID. You have to use this Service ID when you want to use the service.

Reviewer: ```suggestion
This will give the id of your service. You need to use this id whenever you want to use the service.
```
